container_pipeline/lib/command.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+import subprocess
+
+
+def run(cmd):
```

Comment:
Can we name it `run_cmd` or `run_command` to separate it from `run` method we're having in workers as well?